workerapi/server/server.go

```diff
@@ -83,8 +86,7 @@ func (h *handler) updateTimeLastRpc() {
 	h.mu.Unlock()
 }
 
-// Implement thrift worker.Worker interface.
-//
+// Implements thrift worker.Worker interface
```

Comment:
For these comments, it might be better to say "implement worker.Worker.QueryWorker" or even better a summary of what it is. (If that's too much work for this awesome revamp, then maybe TODO?)